_includes/pubbliche-amministrazioni.html

```diff
@@ -83,6 +83,29 @@ <h5 class="card-title text-primary mt-4">
       </div>
     </div>
   </div>
+  <div class="section section-firma section--wpointers">
+    <div class="container">
+      <div class="row py-5 px-3 px-md-0">
+        <div class="col-lg-6 align-self-center order-1 order-md-0">
+          <div>
+            <span class="badge new text-uppercase text-black">novità</span>
+            <h2 class="mt-4 text-white">La firma elettronica qualificata su IO</h2>
+            <p class="mt-4 text-white">Firma con IO è la funzionalità integrata nell’app IO che consente ai tuoi utenti di firmare digitalmente i documenti della Pubblica Amministrazione in modo facile, veloce e sicuro. Tutto, senza uscire dall’app.</p>
+            <a
+              href="https://firma.io.italia.it/"
+              target="_blank"
```

Comment:
add a title attribute with a text to help accessibility (ex. "Visita il sito web")